writings/updated/02-chapter-1.adoc

```diff
@@ -330,7 +330,7 @@ For even the devils believe the holiness that was in Christ, and can preach it,
 but they cannot inherit it, because they do not love to live the life of it.
 
 Hereby I know these two faiths clearly distinguished:
-The end of the one is the life of Christ in which He Himself walked,
+the end of the one is the life of Christ in which He Himself walked,
```

Comment:
The capital `T` seemed like a mistake to me when reading it, but I could also be persuaded that we should keep it that way if it was on purpose. 🤔

Reply:
Shown in context:

![2018-10-31_13-34-52](https://user-images.githubusercontent.com/7050938/47807794-57165a80-dd13-11e8-9053-cfb3550973bd.png)

Reply:
The rule (which almost nobody follows) is - "in American English, the first word after a colon is sometimes capitalized if it begins a complete sentence." I could go either way on this. Does it look weird to you?